README.md

```diff
@@ -136,14 +136,27 @@ PostDuplicateTagValues - Controls when datapoints are logged.  If set to true, d
 
 CommunicationType - This refers to if you are using OMF (OSIsoft message format). If you have PIWEBAPI 2019 or later, and you have OMF enabled, set this string to "omf". If you are using an older version of PIWEBAPI set the string to "piwebapi". Any other values will result in an error. For more information on OMF read the section here:  [OMF](#OMF-Support)
 
-### TagList
-
-TagList - List of eWON tags that should be connected to the OSIsoft PI server.  If PI Points with (non case sensitive) matching names do not exist on the PI server they will be created automatically.
-
 ## Customizing the application
 
 If you wish to modify, debug, or rebuild the application the toolkit and documentation is available here https://developer.ewon.biz/content/java-0. The instructions for setting up your development environment are here  https://developer.ewon.biz/system/files_force/AUG-072-0-EN-%28JAVA%20J2SE%20Toolkit%20for%20eWON%20Flexy%29.pdf?download=1
 
+## Tag Configuration
+To Have a tag's historical data sent to OSIsoft the tag needs to have historical logging enabled and also be assigned to a group. Follow the below steps for each tag.
+
+Navigate to the tag values page. If the "mode" slider on the top left of the screen is set to "view", click on it to set it to "setup".
```

Comment:
It might be neater to enclose text in `blocks`. See https://github.com/hms-networks/IgnitionEwonConnector#setup-historical-logging-for-tags